lib/server/controllers/detectors.controller.js

```diff
@@ -23,7 +24,7 @@ const { updateExpressResponseFromNativeError } = require('../express/updateExpre
 exports.listAllDetectors = async (_request, response) => {
     try {
         const detectors = await detectorService.getAll();
-        response.status(200).json({ data: detectors });
+        response.status(200).json({ data: detectors.filter(({ type }) => [DetectorType.PHYSICAL, DetectorType.VIRTUAL].includes(type)) });
```

Comment:
I suppose here you can use the constant?

Reply:
applied